.github/CONTRIBUTING.md

```diff
@@ -57,6 +57,7 @@ Follow these steps to backport a change into a release branch in compliance with
 3. Create a branch for your change, e.g. `git checkout -b backport`.
 4. When the reason to backport is not obvious from the original commit message, use `git cherry-pick -xe <original commit>` and add a reason. Otherwise use `git cherry-pick -x <original commit>`. That's fine for minor version updates that only include security and bug fixes, commits that fixes an otherwise broken package or similar. Please also ensure the commits exists on the master branch; in the case of squashed or rebased merges, the commit hash will change and the new commits can be found in the merge message at the bottom of the master pull request.
 5. Push to GitHub and open a backport pull request. Make sure to select the release branch (e.g. `release-20.09`) as the target branch of the pull request, and link to the pull request in which the original change was comitted to `master`. The pull request title should be the commit title with the release version as prefix, e.g. `[20.09]`.
+6. When the backport pull request is merged and you have the necessary privileges you can also replace the label `9.needs: port to stable` with `8.has: port to stable` on the original pull request. This way maintainers can keep track of missing backports easier.
```

Comment:
I've always been unclear on this: do we label the original PR with `needs` and the backport PR with `has`? Or do we label the original PR `needs` until a backport PR has been opened and merged, at which point we change it to `has`? I've seen both (and indeed used both).

Reply:
Good question, I got confused about this recently as well. A [quick scan of the first 10 pages](https://github.com/NixOS/nixpkgs/issues?page=1&q=is%3Aclosed+label%3A%228.has%3A+port+to+stable%22) suggests that more than 90% of the time they are used like proposed; The backport PR doesn't receive any additional labels.

I'd also prefer it that way. There is no need to keep track of the state of a backport PR and at the same time it is important to have two bits of information on PRs to unstable. One to indicate that it needs to be backported and the second to distinguish those that got successfully backported from those that accidentally didn't receive the first label.